great_expectations/expectations/registry.py

```diff
@@ -9,7 +9,6 @@
 logger = logging.getLogger(__name__)
 
 _registered_expectations = {}
-_registered_data_assistants = {}
```

Comment:
Purpose of refactor - Let's limit the amount of logic in the global scope. This can be encapsulated quite well in a data assistant-specific class.